annotationprocessingtesting-kt/src/main/java/com/tickaroo/tikxml/annotationprocessing/attribute/Item.kt

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2015 Hannes Dorfmann
+ * Copyright (C) 2015 Tickaroo, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.tickaroo.tikxml.annotationprocessing.attribute
+
+import com.tickaroo.tikxml.annotation.Attribute
+import com.tickaroo.tikxml.annotation.Xml
+import com.tickaroo.tikxml.annotationprocessing.DateConverter
+import java.util.Date
+
+/**
+ * @author Hannes Dorfmann
+ */
+@Xml
+class Item {
+    @Attribute
+    @JvmField
```

Comment:
Did the auto java-->kotlin conversion introduce the `@JvmField`Annotation?
Otherwise, nice catch 👍  to also put that under test!

Reply:
I made that change by hand. Without it will be only testing gettters/setters.

Reply:
Awesome! Great catch! 👍